bcex/examples/simple_strategy.py

```diff
@@ -0,0 +1,199 @@
+import logging
+from datetime import datetime
+
+import pandas as pd
+import pytz
+from bcex.core.orders import OrderSide, OrderType
+from bcex.core.websocket_client import Environment
+from bcex.examples.trader import BaseTrader
+from requests import get
+
+
+def datetime2unixepoch(dt):
+    """ Utility to transform a datetime instance into an unix epoch represented as an int.
+
+    Parameters
+    ----------
+    dt: datetime
+
+    Returns
+    -------
+    int
+        unix epoch
+    """
+    if dt.tzinfo is None:
+        dt = pytz.UTC.localize(dt)
+    else:
+        dt = dt.astimezone(pytz.UTC)
+
+    epoch = pytz.UTC.localize(datetime.utcfromtimestamp(0))
+    delta = dt - epoch
+
+    result = delta.total_seconds()
+    result *= 1000
+
+    return int(result)
+
+
+def unixepoch2datetime(unixepoch):
+    """Given a timestamp, it returns a datetime object.
+
+    Parameters
+    ----------
+    unixepoch : int
+       Timestamp. It can have either a second or millisecond precision
+
+    Returns
+    -------
+    time : datetime
+       Datetime object corresponding to the original timestamp
+    """
+    unixepoch = float(unixepoch) / 1000.0
+
+    time = pytz.UTC.localize(datetime.utcfromtimestamp(unixepoch))
+
+    return time
+
+
+class SimpleStrategy(BaseTrader):
+    def __init__(
+        self,
+        symbol,
+        start_date,
+        rolling_window=30,
+        balance_percentage=0.1,
+        granularity=3600,
+        refresh_rate=60,
+        env=Environment.STAGING,
+        **kwargs,
+    ):
+        channel_kwargs = {"prices": {"granularity": granularity}}
+        super().__init__(
+            symbol,
+            refresh_rate=refresh_rate,
+            env=env,
+            channels_kwargs=channel_kwargs,
+            **kwargs,
+        )
+        self.rolling_window = rolling_window
+        self.granularity = granularity
+        self._historical_candles = None
+        self.start_date = start_date
+        self.balance_percentage = balance_percentage
```

Comment:
you are using it as a fraction?

Reply:
ye to place a % of the relevant balance? can change it if we want

Reply:
I think he means you should rename into `balance_fraction`